source/expectations/matchers/ut_equal.tpb

```diff
@@ -172,6 +172,18 @@ create or replace type body ut_equal as
     return l_result;
   end;
 
+  member procedure include(self in ut_equal, a_items varchar2) is
+  begin
+    include( ut_varchar2_list( a_items ) );
+  end;
+
+  member procedure include(self in ut_equal, a_items ut_varchar2_list) is
+    l_result ut_equal := self;
+  begin
+    l_result.options.include.add_items(a_items);
+    l_result.expectation.to_(l_result );
```

Comment:
I think we need some sort of exception handling.
The syntax that is will work perfectly well are:
- `ut.expect( l_refcursor ).to_( equal(l_refcursor).include( ... ) );`
- `ut.expect( l_refcursor ).to_equal(l_refcursor).include( ... );`

It is however possible to call:
`exec equal(l_refcursor).include( ...);`
In this case, the code will fail dramatically even though it is possible to make a call like this.

Instead of throwing an un-handled exception we should throw a user-friendly one.
Something like:
`ORA-20000 - utPLSQL matcher can only be called as part of expectation. Instead of calling equal( ... ).include( ... ); , call ut.expect( ... ).to_( equal( ... ).include( ... ) );`

Not sure what exact error should be but definitely something that can help users understand their mistake.

Reply:
We can deal with this later.